src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -93,7 +94,7 @@ public virtual IEnumerable<ICompletedFakeObjectCall> RecordedCallsInScope
             get { return FakeScope.Current.GetCallsWithinScope(this); }
         }
 
-        internal List<ICompletedFakeObjectCall> AllRecordedCalls
+        internal IEnumerable<ICompletedFakeObjectCall> AllRecordedCalls
         {
             get { return this.recordedCallsField; }

```

Comment:
> We had been returning the field directly, and @adamralph still suggested the property→method change. Or am completely missing your point?

Perhaps I misread what @adamralph was saying. My understanding was that it would have made sense to make it a method because we would have returned a different list every time. But now that it's just a wrapper for the field, it doesn't really matter. But if you want to make it a method, I don't mind :wink: